### PR TITLE
8-1-updates

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-july-2024.adoc [hunk 1]
```diff
@@ -36,14 +36,14 @@ tt:[24.7.3]
...
-|*Multicloud Support for Last Access Detection*
+|*Support for Last Access Detection*
 
 //IVG-15363/IVG-15364/IVG-15365
 tt:[Secure the Infrastructure]
 
 tt:[24.7.3]
 
-|Prisma Cloud's IAM functionality now includes the ability to to detect inactive cloud service accounts, by querying the days passed since the last usage of the cloud service account. Available for AWS, Azure, and GCP, this RQL clause allows you to determine the days passed since last access. Add the RQL clause below to find last login information for AWS roles, Azure service principals or GCP service accounts.
+|Prisma Cloud's IAM functionality now includes the ability to to detect inactive cloud service accounts, by querying the days passed since the last usage of the cloud service account. Available for AWS and Azure, you can use the RQL clause below to find last login information for AWS roles and Azure service principals.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-july-2024.adoc:46]: Remove the duplicate "to"

File: docs/en/enterprise-edition/content-collections/runtime-security/vulnerability-management/registry-scanning/configure-registry-scanning.adoc [hunk 2]
```diff
@@ -43,6 +43,7 @@ As the scanning takes place, the *Scan status* shows one of the following estate
 +
 * *Scanning*: A deployed Defender is currently scanning the registry.
 * *Pending*: The scanning will take place as soon as a deployed Defender becomes available.
+* *Partially Completed*: Indicates that at least one image has been successfully scanned in the registry. Also displays the number of images that have been scanned successfully.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/runtime-security/vulnerability-management/registry-scanning/configure-registry-scanning.adoc:46]: Add a comma after "Also",

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-07.adoc [hunk 3]
```diff
@@ -82,7 +82,7 @@ You can upgrade the Prisma Cloud Console from a major release to the next major
 //CWP-57783
 |*Severity Mapping Update for Intelligence Stream*
 
-|The severity values for the Intelligence stream are now mapped into 4 predefined values: “Low”, “Medium”,“ High”, and “Critical”. 
+|The severity values for the Intelligence stream are now mapped into 2 predefined values: “ High” and “Medium”.
```
Comment (Aishbn) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-07.adoc:85]: Replace "High" and "Medium" with *High* and *Medium*

**Review by Aishbn — COMMENTED** [review]
Added review comments

**Comment by jenjoe22** [comment]
Closing due to merge conflict... changes to be ported to another PR.

**Pull request closed without merging**